### ISSUE TITLE
illegal character U+005F '_'

### ISSUE THREAD

**Issue by tozhengxq** [github/gh-ost#198]
when some config items contains `'_'` (like "sync_binlog = 1")in the conf file  (conf="/tmp/my1.cnf"),it will error  and stack print:

```
2016-08-25 17:19:26 FATAL /tmp/my1.cnf:62:4: illegal character U+005F '_'
goroutine 1 [running]:
runtime/debug.Stack(0x0, 0x0, 0x0)
       	/usr/local/go/src/runtime/debug/stack.go:24 +0x80
runtime/debug.PrintStack()
       	/usr/local/go/src/runtime/debug/stack.go:16 +0x18
github.com/github/gh-ost/vendor/github.com/outbrain/golib/log.logErrorEntry(0x0, 0x7feaa45a78d0, 0xc820010ca0, 0x0, 0x0)
       	/Users/zhengxq/go/src/github.com/github/gh-ost/vendor/github.com/outbrain/golib/log/log.go:178 +0x159
github.com/github/gh-ost/vendor/github.com/outbrain/golib/log.Fatale(0x7feaa45a78d0, 0xc820010ca0, 0x0, 0x0)
       	/Users/zhengxq/go/src/github.com/github/gh-ost/vendor/github.com/outbrain/golib/log/log.go:255 +0x40
main.main()
       	/Users/zhengxq/go/src/github.com/github/gh-ost/go/cmd/gh-ost/main.go:161 +0x139b
```

**Comment by shlomi-noach** [comment]
Fixed in https://github.com/github/gh-ost/pull/199

**Comment by shlomi-noach** [comment]
The issue should only be closed once the PR is merged :)

**Comment by shlomi-noach** [comment]
Closed by #199

**Comment by shlomi-noach** [comment]
released in https://github.com/github/gh-ost/releases/tag/v1.0.14
